challenges/extreme-constructor-parameter/question.py

```diff
@@ -0,0 +1,31 @@
+"""
+TODO:
+
+Define a decorator `constructor_parameter` that input target class 
+and return a wrapper function with the same signature as the constructor of the target class, 
+and function which decorated by `constructor_parameter` can be called with the instance of the target class.
+"""
+
+class Foo:
+    def __init__(self, a: int, b: str) -> None:
+        ...
+
+def constructor_parameter():
+    ...
+
+
+def should_pass():
+    @constructor_parameter(Foo)
```

Comment:
We may not need to pass in `Foo` in the decorator, since it is given as the function input